zipline/data/data_portal.py

```diff
@@ -35,16 +35,17 @@
 
 log = Logger('DataPortal')
 
-BASE_FIELDS = {
-    'open': 'open',
-    'open_price': 'open',
-    'high': 'high',
-    'low': 'low',
-    'close': 'close',
-    'close_price': 'close',
-    'volume': 'volume',
-    'price': 'close'
-}
+BASE_FIELDS = frozenset([
+    "open", "high", "low", "close", "volume", "price", "last_traded"
```

Comment:
yeah.  the new API uses "open" and "close".

BarData will do the translation from `open_price` to `open` and `close_price` to `close` to support the old API.